Tensile/Common.py

```diff
@@ -876,6 +876,18 @@
     # -3 : Only allow min(GLVWA,GLVWB) < VW ?
     "DepthU":                     depthUs,
 
+    # DepthULdsDivisor determines how we pipeline the data from global memory to LDS
+    # Instead of moving all in-flight data from the register buffer (G2L) to the LDS at once, we divide the G2L buffer into N portions and
+    # write each portion of the G2L to LDS, read from LDS and do the actual matrix multiply-accumulate, before moving on to the portion and so on.
+    # This helps cut down LDS usage by the value of the divisor. Helps increase CU occupancy or DepthU if kernel was previously LDS limited.
+    #
+    # The premise of this parameter is to be able to fetch all 256B (equivalent to 128 half's or 64 single's) in a TN laid-out problem size,
+    # maximizing L2 channel efficiency, therefore this parameter only works for TN buffer layout
+    #
+    # Implementation-wise, for now it only supports ScheduleIterAlg=3 and TransposeLDS=1
+    # In addition, it does not work with DirectToLds=1 because it needs the in-flight data to reside in registers
+    "DepthULdsDivisor":           [1, 2], # [4, 8] not tested yet
```

Comment:
other usage model for this feature is to fetch larger DepthU or MT than LDS size permits. do we have register limit  + LDS size check implemented that checks total size required < 128 KB.

Reply:
It's broken into same 2 separate checks as it was before: 

* Register limit are checked in `KernelWriter::functionSuffix()` due to the dynamics of temporary use of sgpr/vgprs.
* LDS limit is checked in `Solution` constructor, where the LDS is evaluated based on the internal value `DepthULds`

For example, it is possible to generate MT256x128x128 hgemm kernel w/ `DepthULdsDivisor=2`. The unrolled loop would need only (DepthU/2)*(256+128)*bpe ~= 49K LDS size (excluding LDS padding).